symphony/content/content.publish.php

```diff
@@ -1775,19 +1778,21 @@ private function prepareAssociationsDrawer($section)
                     $header = new XMLElement('header');
 
                     // Get the search value for filters and prepopulate
-                    $entry = current(EntryManager::fetch($entry_id));
-                    $search_value = $relation_field->fetchAssociatedEntrySearchValue(
-                        $entry->getData($as['parent_section_field_id']),
-                        $as['parent_section_field_id'],
-                        $entry_id
-                    );
-                    if (is_array($search_value)) {
-                        $search_value = $entry_id;
+                    $filter      = '';
+                    $prepopulate = '';
+                    if ($entry = current(EntryManager::fetch($entry_id))) {
+                        $search_value = $relation_field->fetchAssociatedEntrySearchValue(
+                            $entry->getData($as['parent_section_field_id']),
+                            $as['parent_section_field_id'],
+                            $entry_id
+                        );
+                        if (is_array($search_value)) {
+                            $search_value = $entry_id;
+                        }
+                        $filter      = '?filter[' . $relation_field->get('element_name') . ']=' . $search_value;
+                        $prepopulate = '?prepopulate[' . $as['child_section_field_id'] . ']=' . $search_value;
```

Comment:
Is this alignment recommended by PSR ? IHMO, it just makes the line longer

Reply:
PSR has no opinion about vertical alignment. Personally, I find it more readable this way, separates the variables and the values more clearly.

Reply:
Hum, I'd have to disagree on that. But in the current codebase, I think we do not use this more than we use it.

Reply:
I personally use this alignment style only if I have more consecutive lines of parameters. If it's only 2 lines like here, I wouldn't do it. But that's just my personal taste. And if PSR has no opinion about it…

Reply:
Done.

Reply:
Thanks.